packages/manager/modules/pci/src/components/project/warning-message/warning.constants.js

```diff
@@ -0,0 +1,9 @@
+export const WARNING_DATE = {
+  CREATION_DATE: '01/02/2022',
```

Comment:
CREATION_DATE seems to be higher than LAST_DATE. Is it normal?

Reply:
Creation date here means `DB creation date` and `last date` is last date is the date for customers to cancel their db.

Have renamed the constants